scriptmodules/emulators/jzintv.sh

```diff
@@ -61,17 +70,60 @@ function install_jzintv() {
 }
 
 function configure_jzintv() {
-    mkRomDir "intellivision"
+    local system="intellivision"
```

Comment:
No need to make `system` a variable, since it's not changed.

Reply:
It is more a DRY approach. I don't like to maintain the same information on n locations. But I can remove it or set at least readonly. What is expected the "RetroPie-way"?

Reply:
Leave it as-is then. A variable is used if the emulator supports multiple systems, but it's not the case here and I'd like that `intellivision` is explicit for both `addSystem` and `addEmulator.`

Reply:
Ok. I have put it as it was before the PR.